### ISSUE TITLE
Night sometimes begins before midnight

### ISSUE THREAD

**Issue by GraffJosh** [JPHutchins/open-energy-view#36]
The night slider isn't allowed to start before 12am; recommendation to allow it to be arbitrary.

**Comment by JPHutchins** [comment]
Yes this is a good idea.  Technically what you want is already possible:

* remove the Night time period
* add a new time period after Evening and name it Night

![image](https://user-images.githubusercontent.com/34154542/149451198-f303996c-9b75-4742-b904-d4d84a4fd1ef.png)

From a UI perspective what I think would be awesome is a circular interface!

The interface that exists depends on: https://github.com/sghall/react-compound-slider
